pattern-library/js/select-replace.js

```diff
@@ -12,18 +12,16 @@ define([
      * Choosing an option in the replacement menu also updates the
      * default select menu thus maintaining accessibility.
      */
-
-
     var CustomSelectReplacement = {
 
         vars: {
-            replaced:       $('.replace-select'),
-            replacedClass:  'is-replaced is-transparent',
-            customClass:    'wrapper-custom-select',
-            wrapperClass:   'wrapper-replace-select',
-            valueClass:     'replace-value',
-            iconClass:      'icon-caret-down',
-            hoverClass:     'is-hover'
+            replaced: $('.replace-select'),
```

Comment:
Object keys that have jQuery objects as values can but don't have to start with a `$` due to the `ignoreProperties` flag we set in [the eslint config for this rule](https://github.com/edx/eslint-config-edx/blob/master/README.md#dollar-sign).